response_operations_ui/controllers/message_controllers.py

```diff
@@ -0,0 +1,39 @@
+import jwt
+import logging
+import requests
+from structlog import wrap_logger
+
+from flask import current_app
+from requests.exceptions import HTTPError
+
+from response_operations_ui.exceptions.exceptions import ApiError
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_message_list():
+    logger.debug("Retrieving Message list")
+
+    url = f'{current_app.config["BACKSTAGE_API_URL"]}/v1/secure-message/messages'
+    # This will be removed once UAA is completed.  For now we need the call to backstage to include
+    # an Authorization in its header a JWT that includes party_id and role.
+    encoded_jwt = jwt.encode({'party_id': 'BRES', 'role': 'internal'}, 'testsecret', algorithm='HS256')
```

Comment:
When we start properly encoding JWT payloads, this will need to handle a number of possible exceptions. From looking at the JWT docs, TypeError and ValueError (from the mapping of the JSON) are the two obvious ones.

Reply:
Agreed, but does this need to be done now?